tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: set authorized keys
+  authorized_key:
+    user: "{{ passwordless_ssh_user }}"
+    state: present
+    key: "{{ lookup('file', passwordless_ssh_public_key) }}"
+  become: "{{ passwordless_ssh_become }}"
+  become_user: "{{ passwordless_ssh_become_user }}"
+
+- name: copy private key to remote SSH dir
+  copy:
+    src: "{{ passwordless_ssh_private_key }}"
+    dest: "{{ passwordless_ssh_user_home }}/.ssh/\
+           {{ passwordless_ssh_remote_key_file }}"
+    mode: 0600
+    owner: "{{ passwordless_ssh_user }}"
+  become: "{{ passwordless_ssh_become }}"
+  become_user: "{{ passwordless_ssh_become_user }}"
+
+- name: update known hosts with the other hosts in this current group
+  shell: ssh-keyscan {{ hostvars[item]['ansible_host'] |
+         default(hostvars[item]['inventory_hostname']) }} >>
+         {{ passwordless_ssh_user_home }}/.ssh/known_hosts
+  with_items: "{{ play_hosts }}"
+  become: "{{ passwordless_ssh_become }}"
+  become_user: "{{ passwordless_ssh_become_user }}"
+  register: shell_result
+  changed_when: false
+
+- debug: var=shell_result
```

Comment:
Was this left in for development?  If not, name the task and use indent+colon syntax.

Reply:
This was left in because the source material likes to have this scattered throughout at the end of each such task. Also, I tend to leave names off of debug statements because that's kinda redundant IMO. Is that something we're OK with?

Reply:
Source material?  It seems that ansible-lint contains a special exception for unnamed debug tasks, so it's probably cool.  I might continue to name my debug tasks just because of habit/consistency.